pkg/crc/cache/cache.go

```diff
@@ -130,3 +135,13 @@ func (c *Cache) CheckVersion() error {
 	}
 	return nil
 }
+
+func IsTarball(filename string) bool {
+	tarballExtensions := []string{".tar", ".tar.gz", ".tar.xz", ".zip", ".crcbundle"}
+	for _, extension := range tarballExtensions {
+		if strings.HasSuffix(path.Base(filename), extension) {
```

Comment:
I would do a case-insensitive comparison. Maybe add tar.bz2 to the list of archives? (I forgot if we handle this or not for now).

Reply:
Wondering if rather than detecting tarball vs uncompressed, we should not be making sure that all the binaries we get are compressed?

Reply:
> I would do a case-insensitive comparison.
Sure we can add it.

>  Maybe add tar.bz2 to the list of archives? (I forgot if we handle this or not for now).

No, atm we don't handle it as part of extract.